dev-starts3.sh

```diff
@@ -184,18 +199,22 @@ then
     fake_params+=" --fake_motr_writeobj true --fake_motr_readobj true"
 fi
 
-callgraph_cmd=""
+valgrind_cmd=""
 if [ $callgraph_mode -eq 1 ]
 then
-    callgraph_cmd="valgrind -q --tool=callgrind --collect-jumps=yes --collect-systime=yes --callgrind-out-file=$callgraph_out"
+    valgrind_cmd="valgrind -q --tool=callgrind --collect-jumps=yes --collect-systime=yes --callgrind-out-file=$callgraph_out"
+fi
+if [ $valgrind_memcheck -eq 1 ]
+then
+    valgrind_cmd+="valgrind -q --tool=memcheck --leak-check=full --show-leak-kinds=all --track-origins=yes --verbose --verbose --read-var-info=yes --read-inline-info=yes --log-file=$valgrind_memcheck_out --error-limit=no"
```

Comment:
Will this cmd+= approach work fine if user specifies both callgraph and memcheck?

So either make sure they are mutually exclusive, or -- write code to properly merge valgrind cmd.

Reply:
done